Connectors/Revit/Speckle.Connectors.RevitShared/Operations/Send/Filters/RevitViewsFilter.cs

```diff
@@ -27,12 +26,14 @@ public RevitViewsFilter() { }
   public RevitViewsFilter(RevitContext revitContext)
   {
     _revitContext = revitContext;
-    _doc = _revitContext.UIApplication?.ActiveUIDocument?.Document;
-
-    GetViews();
```

Comment:
Ah... I see the problem this caused before....

Tbh there's a lot about this class I think is quite fragile.
Doing work in a constructor that assumes things about the state of the Revit context... and then providing a very loose contract to consumers is a recipe for disaster.

Reply:
I agree but can't change it now

Reply:
agreed 😁